tools/monorepo-utils/src/code-freeze/commands/milestone/index.ts

```diff
@@ -32,19 +43,18 @@ export const milestoneCommand = new Command( 'milestone' )
 		'-m --milestone <milestone>',
 		'Milestone to create. Next milestone is gathered from Github if none is supplied'
 	)
-	.action( async ( options: Options ) => {
+	.action( async ( options ) => {
 		const { owner, name, dryRun, milestone } = options;
 		const isGithub = getEnvVar( 'CI' );
 
 		if ( milestone && isGithub ) {
 			Logger.error(
 				"You can't manually supply a milestone using Github mode. Please use the CLI locally to add a milestone."
 			);
-			process.exit( 1 );
 		}
 
-		let nextMilestone;
-		let nextReleaseVersion;
+		let nextMilestone = '';
```

Comment:
This can be undefined in some circumstances so init to empty string to satisfy callers that expect a string